endpointmanager/pkg/chplquerier/vendorquerier.go

```diff
@@ -77,7 +78,12 @@ func getVendorJSON(ctx context.Context, client *http.Client) ([]byte, error) {
 		return nil, errors.Wrap(err, "error creating CHPL vendor URL")
 	}
 
-	return getJSON(ctx, client, chplURL)
+	// None of the returned errors should break the system, so print a warning instead
+	jsonBody, err := getJSON(ctx, client, chplURL)
+	if err != nil {
+		log.Warnf("Got error:\n%s\n\nfrom URL: %s", err.Error(), chplURL.String())
+	}
+	return jsonBody, err
```

Comment:
Similar question here, why does this return an error while getProductJSON returns nil?

Reply:
same answer as the above comment